package.json

```diff
@@ -38,7 +38,8 @@
     "prepare": "husky install",
     "prepare-next-release": "standard-version --prerelease pre --release-as minor --skip.changelog=true --skip.commit=true",
     "prepare-release": "tools/prepare-release.sh",
-    "release": "standard-version"
+    "release": "standard-version",
+    "docs": "cd tools/documentation-generation && yarn && yarn docs && npx prettier --write ../../website/docs/cdktf/api-reference/*.mdx"
```

Comment:
nit: Could be a bit more verbose like "generate-docs" for example. But no hard feelings here.